clients/feeder/feeder.go

```diff
@@ -18,10 +18,14 @@ import (
 	"github.com/NethermindEth/juno/starknet"
 	"github.com/NethermindEth/juno/utils"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
-var ErrDeprecatedCompiledClass = errors.New("deprecated compiled class")
+var (
+	//go:embed testdata
+	testData embed.FS
```

Comment:
I don't think we need it because this data will be part of produced binary (not test one, but the one we use in production and releases)